File: TwitchBot/src/BotUser.java
import com.google.gson.annotations.Expose;

public class BotUser {
	@Expose(serialize = true, deserialize = true)
	String username;
	@Expose(serialize = true, deserialize = true)
	int numRequests;
	@Expose(serialize = true, deserialize = true)
	boolean gaveSpot = false;
	@Expose(serialize = true, deserialize = true)
	boolean mod;
	@Expose(serialize = true, deserialize = true)
	boolean follower;
	@Expose(serialize = true, deserialize = true)
	boolean sub;
	@Expose(serialize = true, deserialize = true)
	int points;
	@Expose(serialize = true, deserialize = true)
	int time;
	@Expose(serialize = true, deserialize = true)
	String rank;
	@Expose(serialize = true, deserialize = true)
	long vipCoolDown;
	@Expose(serialize = true, deserialize = true)
	long gambleCoolDown;
	@Expose(serialize = true, deserialize = true)
	int subCredits;

	public BotUser(String username, int numRequests, boolean mod, boolean follower, boolean regular, int points,
			int time, String rank, long vipCoolDown, long gambleCoolDown, int subCredits) {
		this.username = username;
		this.numRequests = numRequests;
		this.mod = mod;
		this.follower = follower;
		this.sub = regular;
		this.points = points;
		this.time = time;
		this.rank = rank;
		this.vipCoolDown = vipCoolDown;
		this.gambleCoolDown = gambleCoolDown;
		this.subCredits = subCredits;
	}

}

File: TwitchBot/WpfApplication1/BotUser.cs
using System;

public class BotUser 
{
    public String username { get; set; }
    public String rank { get; set; }
    public Boolean follower { get; set; }
    public Boolean sub { get; set; }
    public Boolean receivedFollowPayout { get; set; } = false;
    public Boolean mod { get; set; }
    public Boolean gaveSpot { get; set; } = false;
    public int time { get; set; }
    public int subCredits { get; set; }
    public int points { get; set; }
    public int numRequests { get; set; }
    public long months { get; set; } = 0;
    public long gambleCoolDown { get; set; }
    public long vipCoolDown { get; set; }

    public BotUser(String username, int numRequests, Boolean mod, Boolean follower, Boolean sub, int points,
            int time, String rank, long vipCoolDown, long gambleCoolDown, int subCredits)
    {
        this.username = username;
        this.numRequests = numRequests;
        this.mod = mod;
        this.follower = follower;
        this.sub = sub;
        this.points = points;
        this.time = time;
        this.rank = rank;
        this.vipCoolDown = vipCoolDown;
        this.gambleCoolDown = gambleCoolDown;
        this.subCredits = subCredits;
    }

}